docs/releasing.rst

```diff
@@ -8,51 +8,146 @@ Here we try to keep an up to date record of how Mopidy releases are made. This
 documentation serves both as a checklist, to reduce the project's dependency on
 key individuals, and as a stepping stone to more automation.
 
-#. Update the changelog. Commit it.
 
-#. Bump the version number in ``setup.cfg``. Commit it.
+Releasing extensions
+====================
 
-#. Merge the release branch (``develop`` in the example) into master::
+Extensions that are maintained in the Mopidy organization use a quite
+stream-lined release procedure.
+
+#. Make sure that everything has been merged into the `master` branch on
+   GitHub, and that all CI checks are green.
+
+#. Make any manual tests you feel is required.
+
+#. Bump version in `setup.cfg`, to e.g. `2.0.2`.
+
+#. Commit the bumped version::
+
+    git commit -m "Release v2.0.2"
+
+#. Tag the commit::
+
+    git tag -m "Release v2.0.2" v2.0.2
+
+   It is encouraged to use ``-s`` to sign the tag if you have a GnuPG setup.
+
+#. Push to GitHub::
+
+    git push master --follow-tags
```

Comment:
should be git push **origin** master --follow-tags

Reply:
Fixed!